language/tools/test_generation/src/lib.rs

```diff
@@ -47,10 +49,17 @@ pub fn generate_bytecode(
 pub fn run_generation(iterations: usize) {
     env_logger::init();
 
+    let mut valid_programs: u64 = 0;
     for _ in 0..iterations {
         let module =
-            ModuleBuilder::new(1, Some(Box::new(generate_bytecode))).materialize_unverified();
-        debug!("Module: {:#?}", module);
-        run_verifier(module);
+            ModuleBuilder::new(2, Some(Box::new(generate_bytecode))).materialize_unverified();
+        valid_programs += run_verifier(module);
     }
+
+    info!(
+        "Total valid: {}, Total programs: {}, Percent valid: {:?}",
+        valid_programs,
+        iterations,
+        (valid_programs as f64) / (iterations as f64)
```

Comment:
Are you sure this is going to result in a percentage? If so, please explain in a comment.